### PR TITLE
[Tenant Operator] Add Network Policy use-case

### PR DESCRIPTION
**Pull request by abdulhaseeb2**

### CODE REVIEW

**Comment by stakater-user** [comment]
@abdulhaseeb2 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/25dbc76edc06c037c7ae09818c4a6c2de144f721/checks) has Failed!

File: content/sre/tenant-operator/usecases/configuring-multitenant-network-isolation.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+kind: IntegrationConfig
+metadata:
+  name: tenant-operator-config
+  namespace: stakater-tenant-operator
+spec:
+  openshift:
+    project:
+      labels:
+        stakater.com/workload-monitoring: "true"
+        tenant/network-restriction: "true"
```
Comment (tanalam2411) [inline content/sre/tenant-operator/usecases/configuring-multitenant-network-isolation.md]: Can we create a label based on the Template name, that would be an easy pattern to follow?
Create a template and to enable it just add a label by template name and set it to `true`

 `tenant/network-restriction: "true"` -> `tenant-network-policy: "true"`

File: content/sre/tenant-operator/usecases/configuring-multitenant-network-isolation.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Bill has added a new label `tenant/network-restriction: "true"` for tenant projects, now Tenant-Operator will add that label in all projects of tenants.
+
+Finally Bill creates a `TemplateGroupInstance` which will deploy the network policies using the newly created project label and template.
+
+```yaml
+apiVersion: tenantoperator.stakater.com/v1alpha1
+kind: TemplateGroupInstance
+metadata:
+  name: tenant-network-isolation-policy
```
Comment (tanalam2411) [inline content/sre/tenant-operator/usecases/configuring-multitenant-network-isolation.md]: Can we name `tenant-network-isolation-policy` -> `tenant-network-policy-group`, the name doesn't sound interesting but the pattern is easy to remember.

**Review by tanalam2411 — CHANGES_REQUESTED** [review]
Suggested few changes.

**Comment by stakater-user** [comment]
@abdulhaseeb2 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/3a63ba796c812a7853c9fe7d1ab63a564661776a/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/25dbc76edc06c037c7ae09818c4a6c2de144f721/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/3a63ba796c812a7853c9fe7d1ab63a564661776a/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-260-3a63ba79`

**Review by tanalam2411 — APPROVED** [review]
LGTM

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.195`

**Pull request merged**